invisible_cities/io/dst_io_test.py

```diff
@@ -1,37 +1,26 @@
-import os
-import numpy  as np
-import tables as tb
 import pandas as pd
-from . dst_io              import load_dst
-from numpy.testing import assert_allclose
 
-from .. core.testing_utils    import assert_dataframes_equal
-# from ..types.ic_types      import xy
-# from ..evm.event_model     import Cluster
-# from ..evm.event_model     import Hit
-# from ..evm.event_model     import HitCollection
-# from . dst_io              import hits_writer
-from . dst_io                 import load_dst
-from . dst_io                 import load_dsts
+from ..core.testing_utils import assert_dataframes_close
+from .      dst_io        import load_dsts
 
 
-def test_load_dst(Kr_dst_data):
-    (filename, group, node), df = Kr_dst_data
-    dst = load_dst(filename, group, node)
+def test_load_dst(KrMC_kdst):
+    # The function load_dst is actually used in the fixture
```

Comment:
Maybe `load_dst` should be loaded (again) here even if it slows down the tests a couple hundred microseconds just in case someone changes the fixture?

Reply:
Indeed